liquid_node/backup.py

```diff
@@ -38,6 +38,13 @@ def backup(*args):
         if config.is_app_enabled('dokuwiki'):
             backup_files(dest / 'dokuwiki.tgz', '/bitnami/dokuwiki', [], 'dokuwiki:php')
 
+        if config.is_app_enabled('hypothesis'):
+            collection_dir = dest / "hypothesis"
+            collection_dir.mkdir(parents=True, exist_ok=True)
+            backup_pg(collection_dir / 'hypothesis.pg.sql.gz', 'hypothesis', 'hypothesis', 'hypothesis:pg')
+            backup_collection_es(dest / 'hypothesis', 'hypothesis', '/_h_es')
+            backup_files(collection_dir / 'hypothesis.tgz', '/opt/rabbitmq/', [], 'hypothesis:rabbitmq')
```

Comment:
These files will be corrupted, since the `rabbitmq` process might still be running. It's best to leave them out altogether and avoid the internals of how hypothesis saves its data.

If we somehow need the queue, then we'll add another save/restore pair of functions that use the rabbitmq API or CLI to dump and restore the data.